framework/src/org/checkerframework/framework/util/typeinference/DefaultTypeArgumentInference.java

```diff
@@ -212,14 +210,14 @@ protected AnnotatedTypeMirror getAssignedTo(ExpressionTree expression, Annotated
         Pair<InferenceResult, InferenceResult> argInference = inferFromArguments(typeFactory, afArgumentConstraints, targets);
 
         final InferenceResult fromArgEqualities = argInference.first;  //result 2.a
-        final InferenceResult fromArgSupertypes = argInference.second; //result 2.b
+        final InferenceResult fromArgSubandSupers = argInference.second; //result 2.b
```

Comment:
This looks like only the variable name was changed. Why is it now SubAndSuper types?
Should "clampToLowerBound" or one of the other methods called on fromArgSubandSupers be renamed?